website/docs/quickstart/approve.mdx

```diff
@@ -44,10 +44,11 @@ curl -H "Content-Type: application/json" -c git-proxy-cookie -X POST \
 Using the [cookie](/docs/quickstart/approve#2-authenticate-with-the-api) generated, execute a `GET` request to confirm that your push with `ID` exists in the database:
 
 ```bash
-curl -I -b ./git-proxy-cookie http://localhost:8080/api/v1/push/${ID}
+curl -b ./git-proxy-cookie \
```

Comment:
Why the removal of `-I`?

Reply:
-I returns header only and the implementation returns a ```{"message":"not found"}``` body too, which is not visible to the user if -I is added as parameter. Happy to revert :)

Reply:
No problem, happy to keep 👍